performance_tests/src/aws_encryption_sdk_performance_tests/keyrings/hierarchy_keyring.py

```diff
@@ -0,0 +1,174 @@
+# Copyright Amazon.com Inc. or its affiliates. All Rights Reserved.
+# SPDX-License-Identifier: Apache-2.0
+"""Performance tests for the hierarchy keyring."""
+
+# noqa pylint: disable=wrong-import-order
```

Comment:
Do we need this?

Reply:
pylint was throwing some weird errors and I added a bunch of these to debug and missed it while removing. Removed now. Thanks